src/ethereum/ethereumProofPointIssuer.ts

```diff
@@ -72,7 +72,7 @@ export class EthereumProofPointIssuer {
           ethereumProvider
         )
       : await registryRoot.getRegistry();
-    const httpClient = new RealHttpClient();
+    const httpClient = new CachingHttpClient();
```

Comment:
Are the did:web calls the only use of the HTTP client?

Reply:
Yes